src/main/java/com/google/capstone/data/User.java

```diff
@@ -0,0 +1,32 @@
+package com.google.capstone.data;
+
+import java.util.ArrayList;
+
+public class User {
+  private long id;
+  private String email;
+  private ArrayList<SDK> favorites;
```

Comment:
User class shouldn't contains List<SDK>, method addSDKToFavorites & removeSDKFromFavorites . This should be part of UserDao.